### PR TITLE
chore(OnyxFlyoutMenu): Split list design

### PR DESCRIPTION
**Pull request by JoCa96**
Relates to #706, #883

Split Listbox and FlyoutMenu, so that they can develop independently of each other.
It is necessary for #816 and preparation for #883.

### CODE REVIEW

File: packages/sit-onyx/src/components/OnyxFlyoutMenu/OnyxFlyoutMenu.stories.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { defineStorybookActionsAndVModels } from "@sit-onyx/storybook-utils";
+import type { Meta, StoryObj } from "@storybook/vue3";
+import OnyxFlyoutMenu from "./OnyxFlyoutMenu.vue";
+import { h } from "vue";
+import OnyxFlyoutMenuItem from "../OnyxFlyoutMenu/OnyxFlyoutMenuItem.vue";
+
+const meta: Meta<typeof OnyxFlyoutMenu> = {
+  title: "support/OnyxFlyoutMenu",
```
Comment (MajaZarkova) [inline packages/sit-onyx/src/components/OnyxFlyoutMenu/OnyxFlyoutMenu.stories.ts:8]:
```suggestion
  title: "support/FlyoutMenu",
```
We don't use the Onyx prefix here :)

File: packages/sit-onyx/src/components/OnyxListbox/OnyxListbox.vue
Comment (MajaZarkova) [inline packages/sit-onyx/src/components/OnyxListbox/OnyxListbox.vue:1]: The style for multiselect listbox is messed up
image

FIgma:
image

File: packages/sit-onyx/src/components/OnyxFlyoutMenu/OnyxFlyoutMenu.vue
Comment (MajaZarkova) [inline packages/sit-onyx/src/components/OnyxFlyoutMenu/OnyxFlyoutMenu.vue:1]: Is it intentional that focus-visible and keyboard navigation is not supported?

**Review by MajaZarkova — CHANGES_REQUESTED** [review]

File: packages/sit-onyx/src/components/OnyxListbox/OnyxListbox.vue
Comment (MajaZarkova) [reply packages/sit-onyx/src/components/OnyxListbox/OnyxListbox.vue:1]: I fixed this in another PR :)

File: packages/sit-onyx/src/components/OnyxFlyoutMenu/OnyxFlyoutMenu.stories.ts (hunk 1 above)
Comment (JoCa96) [reply packages/sit-onyx/src/components/OnyxFlyoutMenu/OnyxFlyoutMenu.stories.ts]: fixed

File: packages/sit-onyx/src/components/OnyxFlyoutMenu/OnyxFlyoutMenu.vue
Comment (JoCa96) [reply packages/sit-onyx/src/components/OnyxFlyoutMenu/OnyxFlyoutMenu.vue:1]: yes, thats part of #1047

**Review by MajaZarkova — DISMISSED** [review]

File: packages/sit-onyx/src/components/OnyxListboxOption/OnyxListboxOption.vue
Comment (MajaZarkova) [inline packages/sit-onyx/src/components/OnyxListboxOption/OnyxListboxOption.vue:1]: Density is broken

**Review by MajaZarkova — CHANGES_REQUESTED** [review]

File: packages/sit-onyx/src/components/OnyxListboxOption/OnyxListboxOption.vue
Comment (JoCa96) [reply packages/sit-onyx/src/components/OnyxListboxOption/OnyxListboxOption.vue:1]: fixed :)

**Review by MajaZarkova — APPROVED** [review]

**Pull request merged**